src/pipeline.jl

```diff
@@ -100,8 +100,15 @@ function _process_userrecipe(plt::Plot, kw_list::Vector{KW}, recipedata::RecipeD
     plt.n += 1
     kw[:series_plotindex] = plt.n
 
-    push!(kw_list, kw)
-    _add_errorbar_kw(kw_list, kw)
+    if get(kw, :seriestype, :path) == :bar
```

Comment:
Possibly better to check if there are markers? Compare scatterhist and histogram, which both can have errorbars after @oschulz pr.

Reply:
Can you show me a sensible use case of histograms with errorbars? `histogram(randn(100), yerror = 3)` does not seem to work.

Reply:
That's @oschulz ' domain, apparently they are commonly used in particle physics

Reply:
OK, I was just curious, because I wanted to try out histogram and scatterhist examples with errorbars in order to find the right condition, but I could not find out how to do this with plots.

Reply:
If you can't do this intuitively, there's something wrong with the implementation. There's an example here https://github.com/JuliaPlots/Plots.jl/pull/787#issue-225267081.

Reply:
The command for the "scatterhist" with errorbars there is just
```julia
plot(
    h2_norm_pdf,
    yerror = h2_err_norm,
    seriestype = :scatter,
    # xerror = 0.2,
    yscale = :log10,
)
```
So no histogram / scatterhist. To make this work intuitively for histograms too, we would probably need some special casing in the errorbar recipe.

Reply:
OK. It might still be nice to have a comment from @oschulz ?

Reply:
> @daschw: Can you show me a sensible use case of histograms with errorbars

Error bars are quite frequently uses with histograms to show statistical error estimates. If the error information is important, scatterhists with error bars are very popular in some fields (e.g. particle physics).